zetaclient/chains/evm/observer/v2_inbound.go

```diff
@@ -454,6 +462,14 @@ func (ob *Observer) newDepositAndCallInboundVote(event *gatewayevm.GatewayEVMDep
 		coinType = coin.CoinType_Gas
 	}
 
+	// convert erc20Address to asset in foreign coin store to avoid checksum mismatch
```

Comment:
```suggestion
	// convert erc20Address to asset in foreign coin store to avoid checksum mismatch
	// TODO: remove once the checksum conversion is fixed in the protocol
	// https://github.com/zeta-chain/node/issues/3274
```